tasks/8_factorize-number/factorize-number.js

```diff
@@ -0,0 +1,7 @@
+export function factorialize(number) {
+  if (number === 0) {
+    return 1;
+  } else {
+    return number * factorialize(number - 1);
+  }
+}
```

Comment:
here loop solution can be a bit better

Reply:
yes, true because you can easliy get stack overflow with recursive solution